Classes/Hooks/DatamapDataHandlerHook.php

```diff
@@ -106,6 +106,15 @@ private function logStorageError(DataHandler $tceMain, int $storageId, string $m
      */
     protected function getLanguageService(): LanguageService
     {
-        return $GLOBALS['LANG'];
+        $typo3Branch = class_exists(\TYPO3\CMS\Core\Information\Typo3Version::class)
```

Comment:
In https://github.com/mehrwert/TYPO3-FAL-Quota/pull/23/files#diff-a44b53deea21aae003da77ddd86acf1800746018dd2394a982479f42513d703dR165 we use `if (VersionNumberUtility::convertVersionNumberToInteger(TYPO3_version) >= 10000000) {` – would prefer this for consistency here also...